libs/external-router/driver/magento/src/model/magento-url-resolver.ts

```diff
@@ -0,0 +1,5 @@
+import { MagentoResolution } from './resolution';
+
+export interface MagentoUrlResolver {
```

Comment:
I would expect this to be named something like: "MagentoUrlResolutionResponse". I don't understand the usage of the word "Resolver" here.

Reply:
311b3d7d